README.md

```diff
@@ -26,7 +23,7 @@ For a quick sample on demonstrating the usage see [Ballerina By Example](https:/
 
 ## Building from the source
```

Comment:
Should we make change this to `Building from The Source` ?

Reply:
Isn't it `Building from the Source`? Should we capitalise the `the` too?

Reply:
Fixed it as `Building from the Source` in 0cd2273